dashboard/test/ui/features/platform/one_trust.feature

```diff
@@ -1,16 +1,65 @@
 @single_session
 Feature: OneTrust integration
+  @eyes
+  Scenario: User sees OneTrust cookie pop-up when self-hosting OneTrust libraries on hourofcode
+    Given I am on "http://hourofcode.com/uk?otreset=true&otgeo=gb"
+    And I open my eyes to test "Hour of code Onetrust pop up"
+    And I wait until element "#onetrust-banner-sdk" is visible
+    And I see no difference for "Onetrust pop up: Hour of Code" using stitch mode "none"
+    And I close my eyes
+
+  @eyes
+  Scenario: User sees OneTrust cookie pop-up when self-hosting OneTrust libraries on code.org
+    Given I create a student named "Alice"
+    Given I am on "http://studio.code.org/home?otreset=false&otgeo=gb"
+    
+    And I open my eyes to test "Code.org Onetrust pop up"
+    And I wait until element "#onetrust-banner-sdk" is visible 
+    And I see no difference for "Onetrust pop up: code.org" using stitch mode "none"
+    And I close my eyes
+
+  Scenario: OneTrust cookie pop-up shows when self-hosting OneTrust libraries on hourofocode
+    Given I am on "http://hourofcode.com/uk?otreset=true&otgeo=gb"
+    And I wait until element "#onetrust-banner-sdk" is visible
+
+  Scenario: OneTrust cookie pop-up shows when self-hosting OneTrust libraries on code.org
+    Given I create a student named "Alice"
+    Given I am on "http://studio.code.org/home?otreset=false&otgeo=gb"
+    And I wait until element "#onetrust-banner-sdk" is visible
+  
+  Scenario: The pages load the self hosted OneTrust libraries.
+    Given I am on "http://studio.code.org/users/sign_in"
+    Then element "script[src$='onetrust/scripttemplates/otSDKStub.js']" does exist
+    Then element "script[src$='977d/OtAutoBlock.js']" does exist
+    Then element "script[src$='977d-test/OtAutoBlock.js']" does not exist
+
+    Given I am on "http://code.org/index"
+    Then element "script[src$='onetrust/scripttemplates/otSDKStub.js']" does exist
+    Then element "script[src$='977d/OtAutoBlock.js']" does exist
+    Then element "script[src$='977d-test/OtAutoBlock.js']" does not exist
+
+    Given I am on "http://hourofcode.com/us"
+    Then element "script[src$='otSDKStub.js']" does exist
+    Then element "script[src$='e345/OtAutoBlock.js']" does exist
+    Then element "script[src$='e345-test/OtAutoBlock.js']" does not exist
+
   Scenario: The pages load the prod OneTrust libraries.
```

Comment:
Since we added "self-hosting" in the scenarios above, maybe we should distinguish these existing tests by saying something like "OneTrust hosted prod libraries"